src/helpers/eventBridge.ts

```diff
@@ -162,6 +168,30 @@ export default class EventBridge {
     return result;
   }
 
+  async getAllEvents(): Promise<EventBridgeEvents> {
+    let allEventsFound = false;
+    const allEventBridgeEvents: EventBridgeEvents = {};
+
+    while (!allEventsFound) {
+      try {
+        const lastEventBridgeEvents = await this.getEvent();
+
+        if (!lastEventBridgeEvents || !lastEventBridgeEvents.Messages) {
+          return allEventBridgeEvents;
+        }
```

Comment:
What are your thoughts on changing this to:

```suggestion
        if (!lastEventBridgeEvents || !lastEventBridgeEvents.Messages) {
          allEventsFound = true;
          break;
        }
```

So that we still exit the loop early but have a single point of return at the end of the function definition?